trinity/nodes/full.py

```diff
@@ -0,0 +1,45 @@
+from evm.chains.base import BaseChain
+from p2p.peer import (
+    HardCodedNodesPeerPool,
+)
+from p2p.server import Server
+from p2p.service import BaseService
+
+from trinity.nodes.base import Node
+from trinity.utils.chains import (
+    ChainConfig,
+)
+
+
+class FullNode(Node):
+    _chain: BaseChain = None
+
+    def __init__(self, chain_config: ChainConfig) -> None:
+        super().__init__(chain_config)
+
+        self._bootstrap_nodes = chain_config.bootstrap_nodes
+        self._network_id = chain_config.network_id
+        self._node_key = chain_config.nodekey
+        self._node_port = chain_config.port
+
+    def get_chain(self):
+        if self._chain is None:
+            self._chain = self.chain_class(self.db_manager.get_db())
+
+        return self._chain
+
+    def get_peer_service(self) -> BaseService:
```

Comment:
The term `peer_service` seems a little off here.  Maybe something like `get_p2p_server` or `get_networking_server`?  Feels like the name `server` should be present.